Sources/VLCSettingsController.m

```diff
@@ -172,4 +174,27 @@ - (void)updateForPasscode:(NSString *)passcode
         [self.navigationController.presentedViewController dismissViewControllerAnimated:YES completion:nil];
     }
 }
+
+#pragma mark - InAppSettings customization
+
+- (CGFloat)settingsViewController:(id<IASKViewController>)settingsViewController tableView:(UITableView *)tableView heightForHeaderForSection:(NSInteger)section
+{
+    return section == 0 ? 0 : SETTINGS_HEADER_HEIGHT;
+}
+
+- (UIView *)settingsViewController:(id<IASKViewController>)settingsViewController tableView:(UITableView *)tableView viewForHeaderForSection:(NSInteger)section
+{
+    if (section == 0) {
+        return nil;
+    }
+    VLCSettingsTableHeaderView *header = [tableView dequeueReusableHeaderFooterViewWithIdentifier:VLCSettingsTableHeaderView.identifier];
+    header.headerLabel.text = [self.settingsReader titleForSection:section];
+    return header;
+}
+
+- (NSString *)tableView:(UITableView *)tableView titleForHeaderInSection:(NSInteger)section
+{
+    return nil;
+}
```

Comment:
This can be deleted it will fall back to viewForHeaderInSection

Reply:
Hmmm without it, the header titles come out capitalized. I think this is because InAppSettings has a default `titleForHeaderInSection`. So this is required to hide `super`'s implementation.